script/DeployAvaFRAXSteward.s.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/console.sol';
+import {Script} from 'forge-std/Script.sol';
+import {AaveV3AvaFRAXListingSteward} from '../src/contracts/frax/AaveV3AvaFRAXListingSteward.sol';
+
+contract DeployAvaFRAXSteward is Script {
+    function run() external {
+        vm.startBroadcast();
+        AaveV3AvaFRAXListingSteward fraxSteward = new AaveV3AvaFRAXListingSteward();
+        console.log('Frax Steward address', address(fraxSteward));
```

Comment:
is this needed? i think broadcast will automatically log all deployed contract addresses

Reply:
True, I was not aware of that before